app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Admin
+  class UsersController < AdminController
```

Comment:
This controller has just been moved into the admin namespace. It's not entirely new.